foundation/foundation-mda/src/main/resources/profiles.json

```diff
@@ -994,6 +994,9 @@
 			},
 			{
 				"name": "hiveMetastoreServiceArgoCD"
+			},
+			{
+				"name": "valuesCIFile"
```

Comment:
Q: This was just accidentally left out?

Reply:
I think so. Otherwise, it will give error when create apps